README.md

```diff
@@ -1,45 +1,47 @@
-# CTI-website-backend
+# CTI-Website-Backend
+## Running Locally 
 
-A Django app, which can be deployed to Heroku.
+- Import data from data source
+    - [Excel Sheet](https://drive.google.com/file/d/1xiLeyMwZc-n6eB1R_XdCJ00YrarfnR_w/view)
+    - Download as csv
+    - Upload the csv file in the project root 
 
-This application supports the [Getting Started with Python on Heroku](https://devcenter.heroku.com/articles/getting-started-with-python) article - check it out.
+Below are the steps to get started using docker and pip
 
-## Running Locally
-
-Make sure you have Python 3.7 [installed locally](http://install.python-guide.org). To push to Heroku, you'll need to install the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli), as well as [Postgres](https://devcenter.heroku.com/articles/heroku-postgresql#local-setup).
-
-```sh
-$ git clone https://github.com/heroku/python-getting-started.git
-$ cd python-getting-started
-
-$ python3 -m venv getting-started
-$ pip install -r requirements.txt
-
-$ createdb python_getting_started
-
-$ python manage.py migrate
-$ python manage.py collectstatic
-
-$ heroku local
+- Create a .env file inside /config for writing the database credentials, example below
```

Comment:
You won't be able to run the project locally (outside of the docker-compose network)-- this requires that an image is built to run your application.

You can spin up docker-compose and expose port 5432 so that it is accessible from outside your docker-compose network can prevent configuration nightmares if someone is trying to run/debug the application using a local python runtime.

Reply:
Possibly use a different port for postgres - preventing local port collisions

Reply:
handled by @shinjonathan